src/ansys/fluent/visualization/pyvista/pyvista_objects.py

```diff
@@ -35,6 +35,52 @@ def __init__(
         )
 
 
+class Contours:
```

Comment:
`class Contour` is already defined therefore we have chosen `Contours` name.

Reply:
@seanpearsonuk  Yes, sure. We haven't decided the exact location yet, hence mentioned the reason. Where we should put this code ?

Reply:
For current purposes, src/ansys/fluent/visualization/contour.py will suffice.

Reply:
Done. Thank you.